hols/todo/README.md

```diff
@@ -0,0 +1,421 @@
+# todo with Helidon and Coherence
+
+## Configure database
+
+### Oracle Database
+1. Create a table
+```
+CREATE TABLE TASKS(ID VARCHAR2(255), TITLE VARCHAR2(255), COMPLETED BOOLEAN)
+```
+2. Download the wallet to a known directory and extract
+3. Create a file `hibernate.cfg.xml` file in a known location
+
+```xml
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE hibernate-configuration PUBLIC
+        "-//Hibernate/Hibernate Configuration DTD 5.3//EN"
+        "http://hibernate.org/dtd/hibernate-configuration-3.0.dtd">
+
+<hibernate-configuration>
+    <session-factory>
+        <property name="hbm2ddl.auto">update</property>
+        <property name="connection.url">jdbc:oracle:thin:@tasksdb_medium?TNS_ADMIN=/wallets/task_db</property>
+        <property name="connection.username">task_db_user</property>
+        <property name="connection.password">changeme</property>
+        <property name="connection.driver_class">oracle.jdbc.OracleDriver</property>
+        <property name="hibernate.oracleucp.connectionFactoryClassName">oracle.jdbc.datasource.impl.OracleDataSource
+        </property>
+
+        <mapping class="io.helidon.labs.todo.coherence.Task"/>
+    </session-factory>
+</hibernate-configuration>
+                            
+## Build modules
+
+### Build everything locally
+ 
+```bash
+mvn clean package
+```   
+ 
+### Build individual module
+
+* coherence
```

Comment:
No need to document individual build, this is not a Maven crash course.

Reply:
ok